test/integration/client-side-encryption/client_side_encryption.prose.22.range_explicit_encryption.test.ts

```diff
@@ -537,7 +537,7 @@ describe('Range Explicit Encryption', function () {
             })
             .catch(e => e);
 
-          expect(resultOrError).to.be.instanceOf(TypeError);
+          expect(resultOrError).to.be.instanceOf(MongoCryptError);
```

Comment:
I don't have a better place to leave a comment, but we wanted to remove the custom matcher isMongoCryptError from the unified spec runner